core/src/main/java/org/apache/accumulo/core/iterators/SortedKeyValueIterator.java

```diff
@@ -147,4 +147,25 @@
    *              if not supported.
    */
   SortedKeyValueIterator<K,V> deepCopy(IteratorEnvironment env);
+
+  /**
+   * Closes the Iterator. This must be overridden by the implementing class that has access to <tt>SortedKeyValueIterator source</tt> provided in the
+   * <tt>init</tt> method.
+   */
+  @Override
+  default void close() throws Exception {
```

Comment:
After some discussion with @ctubbsii and @keith-turner, we were thinking of removing the "throws Exception".  We came to the conclusion that we don't want to assume that the close will throw an exception. This will require removing the IOException throws from FileSKVIterator.close and modifying its implementing classes to handle the IOException.  What do you think @joshelser ?